runtime/CSharp/tests/issue-3510/ChannelTest.g4

```diff
@@ -0,0 +1,7 @@
+grammar ChannelTest;
+
+prog : INT+ EOF;
+INT : [0-9]+;
+AAA : [a-z]+ -> channel(HIDDEN); // Note--lowercase "channel"
+BBB : [A-Z]+ -> Channel(HIDDEN); // Note--uppercase "Channel"
```

Comment:
Not sure it makes sense to use both options for `channel` here. Case insensitivity should be check in another test.